### PR TITLE
Content-Type consistency

### PR DESCRIPTION
**Pull request by earth2marsh**
Needs a second set of eyes!

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -1080,21 +1080,21 @@ content:
...
-<a name="contentTypeExample"></a>example | [Example Object](#exampleObject) | Example of the content type.  The example object SHOULD be in the correct format as specified in the _content_ type.  The `example` object is mutually exclusive to the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value SHALL _override_ the the example provided by the schema.
+<a name="contentTypeExamples"></a>examples | [Examples Array](#examplesArray) | Examples of the Content-Type.  Each example in the Examples array SHOULD be in the correct format as specified in the media type.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value SHALL _override_ the example provided by the schema.
```
Comment (darrelmiller) [inline versions/3.0.md]: I don't really like the use of `Content-Type` here as it makes me think that it is an example of the value in a `Content-Type` header.

File: versions/3.0.md [hunk 2]
```diff
@@ -1080,21 +1080,21 @@ content:
...
+<a name="contentTypeExample"></a>example | [Example Object](#exampleObject) | Example of the Content-Type.  The example object SHOULD be in the correct format as specified in the media type.  The `example` object is mutually exclusive to the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value SHALL _override_ the the example provided by the schema.
```
Comment (darrelmiller) [inline versions/3.0.md]: As above.

Comment (earth2marsh) [inline versions/3.0.md]: works for me. Updated.

**Comment by earth2marsh** [comment]
Addressed @darrelmiller 's comments.

**Comment by darrelmiller** [comment]
I added #943 to rename Content Type Object, so there is a fair amount of overlap with this PR so we need to reconcile the changes in the two if my PR is accepted.

**Comment by darrelmiller** [comment]
These changes were incorporated into PR #943

**Pull request closed without merging**